internal/clicontext/context.go

```diff
@@ -99,3 +106,21 @@ func RepoClientsetFromContext(ctx context.Context) repoclient.RepoClientset {
 	}
 	return nil
 }
+
+// TODO too web specific and should maybe be an web-extension ?
```

Comment:
I think I agree. Maybe we can put this into a new `internal/web/context` package?